systemtests/src/main/java/io/enmasse/systemtest/utils/TestUtils.java

```diff
@@ -350,7 +350,7 @@ public static void waitForNamespaceDeleted(Kubernetes kubernetes, String namespa
             try {
                 log.debug("Running command, attempt: {}", i);
                 return fn.call();
-            } catch (Exception ex) {
+            } catch (Exception | AssertionError ex) {
```

Comment:
Won't this actually mask errors?

Reply:
I don't think so, the reason why I did this, is runUntullPass does not catch exception in method passed into because Exception is not a parent of AssertionError

Reply:
If there is an error and it will still be flagged at the last failure, thats fine I suppose.

Reply:
yes